### PR TITLE
19556 - Account Search for BCOL Account # in BCROS

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/19556

*Description of changes:*
1. Add Account Search for BCOL Account # in BCROS
2. Update StaffActiveAccountsTable.vue to composition-api

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2856-fvweegbg.web.app

File: auth-web/src/components/auth/staff/account-management/StaffActiveAccountsTable.vue [hunk 1]
```diff
@@ -182,244 +182,252 @@
...
+                (params.orgType.length > 0 && params.orgType !== OrgAccountTypes.ALL)
+    }
 
-  protected doSearchParametersExist (searchParams: OrgFilterParams) {
-    return searchParams.name.length > 0 ||
-          searchParams.branchName.length > 0 ||
-          searchParams.id.length > 0 ||
-          searchParams.decisionMadeBy.length > 0 ||
-          (searchParams.orgType.length > 0 && searchParams.orgType !== OrgAccountTypes.ALL)
+    mounted()
```
Comment (seeker25) [inline auth-web/src/components/auth/staff/account-management/StaffActiveAccountsTable.vue:410]: Looks like the difference here between setup and onMounted... 
onMounted has access to the DOMand setup doesn't.

**Review by seeker25 — APPROVED** [review]
Looks great to me Jia.. nice work!

**Pull request merged**